modular_skyrat/modules/company_imports/code/armament_datums/vitezstvi_ammo.dm

```diff
@@ -0,0 +1,130 @@
+	item_type = /obj/item/circuitboard/machine/ammo_workbench
+	lower_cost = CARGO_CRATE_VALUE * 10
+	upper_cost = CARGO_CRATE_VALUE * 20
+	interest_required = PASSED_INTEREST
+	interest_addition = COMPANY_INTEREST_GAIN_BIG
+
+/datum/armament_entry/company_import/vitezstvi/ammo_bench
+	item_type = /obj/item/disk/ammo_workbench/lethal
+	lower_cost = CARGO_CRATE_VALUE * 10
+	upper_cost = CARGO_CRATE_VALUE * 20
+	interest_required = HIGH_INTEREST
+	interest_addition = COMPANY_INTEREST_GAIN_BIG
+
+// Boxes of non-shotgun ammo
+
+/datum/armament_entry/company_import/vitezstvi/ammo_boxes
+	subcategory = "Ammunition Boxes"
+	lower_cost = CARGO_CRATE_VALUE * 0.75
+	upper_cost = CARGO_CRATE_VALUE * 1.25
+
+/datum/armament_entry/company_import/vitezstvi/ammo_boxes/pepperball
+	item_type = /obj/item/ammo_box/advanced/pepperballs
+
+/datum/armament_entry/company_import/vitezstvi/ammo_boxes/m1911_lethals
+	item_type = /obj/item/ammo_box/c45
+
+/datum/armament_entry/company_import/vitezstvi/ammo_boxes/wt550_lethals
+	item_type = /obj/item/ammo_box/c46x30mm
+
+/datum/armament_entry/company_import/vitezstvi/ammo_boxes/wt550_piercing
+	item_type = /obj/item/ammo_box/c46x30mm/ap
+
+/datum/armament_entry/company_import/vitezstvi/ammo_boxes/wt550_rubber
+	item_type = /obj/item/ammo_box/c46x30mm/rubber
+
+/datum/armament_entry/company_import/vitezstvi/ammo_boxes/peacekeeper_lethal
+	item_type = /obj/item/ammo_box/advanced/b9mm
+
+/datum/armament_entry/company_import/vitezstvi/ammo_boxes/peacekeeper_hp
+	item_type = /obj/item/ammo_box/advanced/b9mm/hp
+
+/datum/armament_entry/company_import/vitezstvi/ammo_boxes/peacekeeper_rubber
+	item_type = /obj/item/ammo_box/advanced/b9mm/rubber
+
+/datum/armament_entry/company_import/vitezstvi/ammo_boxes/auto10mm_lethal
+	item_type = /obj/item/ammo_box/advanced/b10mm
+
+/datum/armament_entry/company_import/vitezstvi/ammo_boxes/auto10mm_hp
+	item_type = /obj/item/ammo_box/advanced/b10mm/hp
+
+/datum/armament_entry/company_import/vitezstvi/ammo_boxes/auto10mm_rubber
+	item_type = /obj/item/ammo_box/advanced/b10mm/rubber
+
+/datum/armament_entry/company_import/vitezstvi/ammo_boxes/sabel_lethal
+	item_type = /obj/item/ammo_box/c56mm
+	lower_cost = CARGO_CRATE_VALUE * 1.25
+	upper_cost = CARGO_CRATE_VALUE * 1.5
+
+/datum/armament_entry/company_import/vitezstvi/ammo_boxes/sabel_rubber
+	item_type = /obj/item/ammo_box/c56mm/rubber
+	lower_cost = CARGO_CRATE_VALUE * 1.25
+	upper_cost = CARGO_CRATE_VALUE * 1.5
+
+/datum/armament_entry/company_import/vitezstvi/ammo_boxes/sabel_hunting
+	item_type = /obj/item/ammo_box/c56mm/hunting
+	lower_cost = CARGO_CRATE_VALUE * 1.25
+	upper_cost = CARGO_CRATE_VALUE * 1.5
+
+/datum/armament_entry/company_import/vitezstvi/ammo_boxes/sabel_blank
+	item_type = /obj/item/ammo_box/c56mm/blank
+	lower_cost = CARGO_CRATE_VALUE * 0.1
+	upper_cost = CARGO_CRATE_VALUE * 0.3
+
+// Revolver speedloaders
+
+/datum/armament_entry/company_import/vitezstvi/speedloader
```

Comment:
might want to give it the box treatment instead of a singular speedloader while upping the cost